### PR TITLE
add Swagger spec for Bing Local Search API.

### PR DESCRIPTION
**Pull request by davidbargeron**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json [hunk 1]
```diff
@@ -0,0 +1,932 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Local Search API",
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json]: Please use "Local Search Client".
title will be used as client class name.

Comment (yangyuan) [inline specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json]: Please use "Local Search Client".
title will be used as client class name.

**Review by yangyuan — CHANGES_REQUESTED** [review]
This swagger will not be able to be used for generating SDKs as autorest "readme.md" is missing.

File: specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json [hunk 2]
```diff
@@ -0,0 +1,946 @@
...
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "paths": {
+    "/localbusinesses/v7.0/search": {
+      "get": {
```
Comment (sergey-shandar) [inline specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json]: Please, provide an example for the operation.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json [hunk 3]
```diff
@@ -0,0 +1,946 @@
...
+        },
+        "adaptiveCard": {
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "QueryContext": {
+      "description": "Defines the query context that Bing used for the request.",
+      "discriminator": "_type",
```
Comment (sergey-shandar) [inline specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json]: Could we rename the discriminator? Usually, names should follow `camelCase` style.

File: specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json (hunk 2 above)
Comment (davidbargeron) [reply specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json]: just added x-ms-examples. See commit https://github.com/Azure/azure-rest-api-specs/pull/4349/commits/682b78525dccd7bfd44e60bfaa553a15f166e355.

File: specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json (hunk 3 above)
Comment (davidbargeron) [reply specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json]: QueryContext is an inherited schema that is used all over the place. I cannot change it.

File: specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json (hunk 1 above)
Comment (davidbargeron) [reply specification/cognitiveservices/data-plane/LocalSearch/stable/v1.0/LocalSearch.json]: DONE. See 
https://github.com/Azure/azure-rest-api-specs/pull/4349/commits/8ddfa8da1b8910b62f375dab48b937f5233bc4f6

**Review by sergey-shandar — APPROVED** [review]

**Comment by sergey-shandar** [comment]
@yangyuan is it ok to merge?

**Review by yangyuan — APPROVED** [review]

**Pull request merged**